dev-util/poke/poke-2.4-r1.ebuild

```diff
@@ -42,7 +42,7 @@ RDEPEND="
 "
 DEPEND="
 	${RDEPEND}
-	test? ( nbd? ( sys-libs/libnbd[uri-support] ) )
+	test? ( nbd? ( sys-libs/libnbd ) )
```

Comment:
this should be [uri-support(+)]: https://devmanual.gentoo.org/general-concepts/dependencies/index.html#use-dependency-defaults